README.md

```diff
@@ -1,6 +1,9 @@
 Shx
 ===
 
+[![Travis](https://img.shields.io/travis/shelljs/shx.svg)](https://travis-ci.org/shelljs/shx)
+[![AppVeyor](https://img.shields.io/appveyor/ci/ariporad/shx.svg?maxAge=2592000)](https://ci.appveyor.com/project/ariporad/shx)
```

Comment:
It has to be made by the admin of the "shelljs" account (which I guess is me). I'm actually trying right now to figure out how to decouple "shelljs" from my personal account. Appveyor docs are not very helpful. If you guys can find any info, that would be appreciated.